go.mod

```diff
@@ -0,0 +1,84 @@
+module github.com/tektoncd/triggers
+
+go 1.13
```

Comment:
Hmm, does this need go 1.13? I don't think our test runner works with 1.13 yet

Reply:
Not sure. This was auto generated by the `go mod init`. We can try bumping this down, but I suspect we might be fighting the tool to keep this out.

Any reason we can't bump the test runner to 1.13?

Reply:
looks like @dibyom went ahead and bumped it? https://github.com/tektoncd/plumbing/pull/88 :D :tada:

Reply:
I think it's fine having 1.13 in there :wink: